rs/sns/governance/src/governance.rs

```diff
@@ -3887,14 +3886,19 @@ impl Governance {
 
         let mut swap_neurons = vec![];
 
-        // TODO(NNS1-3198): Simplify this code after `NeuronParameters` is made obsolete.
-        let neuron_recipes_from_new_source = request.neuron_recipes;
+        let Some(neuron_recipes) = request.neuron_recipes else {
+            return ClaimSwapNeuronsResponse::new_with_error(ClaimSwapNeuronsError::Internal);
```

Comment:
Why don't we ass context to this error to say that ".neuron_recipes" must be specified?